docs/src/app/components/PropTypeDescription.jsx

```diff
@@ -26,9 +34,12 @@ const PropTypeDescription = React.createClass({
   render() {
     const {
       code,
+      component,
+      description,
     } = this.props;
 
-    let text = `### Properties
```

Comment:
Oooo, I didn't think of it that way. I like your approach a lot more. I'll do a patch.